hasjob/models/filtered_view.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+from flask import url_for
+from sqlalchemy.dialects import postgresql
+from . import db, BaseNameMixin, JobType, JobCategory, JobLocation, Tag, Domain, User
+
+__all__ = ['FilteredView']
+
+
+filteredview_jobtype_table = db.Table('filteredview_jobtype_table', db.Model.metadata,
+    db.Column('filtered_view_id', None, db.ForeignKey('filtered_view.id'), primary_key=True),
+    db.Column('jobtype_id', None, db.ForeignKey('jobtype.id'), primary_key=True),
+    db.Column('created_at', db.DateTime, nullable=False, default=db.func.utcnow())
+)
+
+
+filteredview_jobcategory_table = db.Table('filteredview_jobcategory_table', db.Model.metadata,
+    db.Column('filtered_view_id', None, db.ForeignKey('filtered_view.id'), primary_key=True),
+    db.Column('jobcategory_id', None, db.ForeignKey('jobcategory.id'), primary_key=True),
+    db.Column('created_at', db.DateTime, nullable=False, default=db.func.utcnow())
+)
+
+
+filteredview_tag_table = db.Table('filteredview_tag_table', db.Model.metadata,
+    db.Column('filtered_view_id', None, db.ForeignKey('filtered_view.id'), primary_key=True),
+    db.Column('tag_id', None, db.ForeignKey('tag.id'), primary_key=True),
+    db.Column('created_at', db.DateTime, nullable=False, default=db.func.utcnow())
+)
+
+filteredview_domain_table = db.Table('filteredview_domain_table', db.Model.metadata,
+    db.Column('filtered_view_id', None, db.ForeignKey('filtered_view.id'), primary_key=True),
+    db.Column('domain_id', None, db.ForeignKey('domain.id'), primary_key=True),
+    db.Column('created_at', db.DateTime, nullable=False, default=db.func.utcnow())
+)
+
+
+class FilteredView(BaseNameMixin, db.Model):
```

Comment:
Filtered views are scoped to board. Needs `BaseScopedNameMixin` and a `board_id` column.